python/generate/templates/model.mustache

```diff
@@ -62,7 +66,7 @@ class {{classname}}(object):
     }
 {{/discriminator}}
 
-    def __init__(self{{#vars}}, {{name}}{{#defaultValue}}={{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}=None{{/defaultValue}}{{/vars}}):  # noqa: E501
+    def __init__(self{{#vars}}, {{name}}={{#defaultValue}}{{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}None{{/defaultValue}}{{/vars}}, local_vars_configuration=None):  # noqa: E501
```

Comment:
I feel that the equals sign was moved and duplicated for a reason here but I can't remember why. Do you know if this doesn't cause any awkward issues having it moved to the front?

Reply:
do you mean this was a change we made or is this from updating the template? tests all pass so I would presume its ok